examples/metrics/src/main/java/io/opentelemetry/example/metrics/DoubleCounterExample.java

```diff
@@ -0,0 +1,65 @@
+package io.opentelemetry.example.metrics;
+
+import io.opentelemetry.OpenTelemetry;
+import io.opentelemetry.context.Scope;
+import io.opentelemetry.metrics.DoubleCounter;
+import io.opentelemetry.metrics.Meter;
+import io.opentelemetry.trace.Span;
+import io.opentelemetry.trace.Span.Kind;
+import io.opentelemetry.trace.Status;
+import io.opentelemetry.trace.Tracer;
+import java.io.File;
+import java.util.ArrayList;
+import java.util.List;
+import javax.swing.filechooser.FileSystemView;
+
+/**
+ * Example of using {@link DoubleCounter} and {@link DoubleCounter.BoundDoubleCounter} to count disk space used by files with a specific extensions.
+ */
+public class DoubleCounterExample {
+
+  private static final Tracer tracer =
+      OpenTelemetry.getTracer("io.opentelemetry.example.metrics");
```

Comment:
```suggestion
      OpenTelemetry.getTracer("io.opentelemetry.example.metrics", "0.5");
```
You would usually supply the same library version as for the meter below.